### ISSUE TITLE
Pino doesn't build with Valac 0.7.10

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#79]
```
1. Install valac 0.7.10 on your system.
2. Launch ./waf configure --prefix=/usr (which will work)
3. Launch ./waf build (whic will fail)

It was working fine with valac 0.7.9, problems started with new version.
Here's error output after ./waf build:

Build failed
 -> task failed (err #1): 
    {task: valac_task
twitter_url.vala,timeline_list.vala,tray_icon.vala,prefs.vala,rest_api_re.vala,d
m_entry.vala,template.vala,hig_table.vala,rest_api_direct.vala,timeline_list_abs
tract.vala,gtk_style.vala,timeline_direct_list.vala,more_window.vala,pref_dialog
.vala,cache.vala,time_utils.vala,popups.vala,rest_api_abstract.vala,main_window.
vala,log_window.vala,rest_url_interface.vala,list_abstract.vala,re_tweet.vala,st
atus_bar_smart.vala,main.vala,rest_api_timeline.vala,userpic_image.vala,timer.va
la
->
twitter_url.c,timeline_list.c,tray_icon.c,prefs.c,rest_api_re.c,dm_entry.c,templ
ate.c,hig_table.c,rest_api_direct.c,timeline_list_abstract.c,gtk_style.c,timelin
e_direct_list.c,more_window.c,pref_dialog.c,cache.c,time_utils.c,popups.c,rest_a
pi_abstract.c,main_window.c,log_window.c,rest_url_interface.c,list_abstract.c,re
_tweet.c,status_bar_smart.c,main.c,rest_api_timeline.c,userpic_image.c,timer.c}

Full output is attached.

Pino 0.1.4 on Debian Sid with valac 0.7.10.
```

Original issue reported on code.google.com by `had...@gmail.com` on 6 Feb 2010 at 1:09

Attachments:
- [pino-valac-error.txt](https://storage.googleapis.com/google-code-attachments/pino-twitter/issue-79/comment-0/pino-valac-error.txt)

**Comment by GoogleCodeExporter** [comment]
```
Issue 80 has been merged into this issue.
```

Original comment by `tro...@gmail.com` on 6 Feb 2010 at 6:25

**Comment by GoogleCodeExporter** [comment]
```
Issue 83 has been merged into this issue.
```

Original comment by `tro...@gmail.com` on 7 Feb 2010 at 2:11

**Comment by GoogleCodeExporter** [comment]
```
Closed by 02b5b99785ee revision.
```

Original comment by `tro...@gmail.com` on 10 Feb 2010 at 6:58
- Changed state: **Fixed**
